src/user/dashboard/news-feed/popups/comment.js

```diff
@@ -0,0 +1,62 @@
+import React, {Component} from 'react';
+import {Modal, Button, Row, Col, Form} from 'react-bootstrap';
+import "./comment.scss"
+import logo from "../../../../svgs/profile-icon.svg";
+
+export class Comment extends Component{
+    // eslint-disable-next-line
+    constructor(props){
+        super(props);
+    }
+    render(){
+    return (
+        <Modal
+      {...this.props}
+      size="md"
+      aria-labelledby="contained-modal-title-vcenter"
+      centered>
+      <div className="container">
```

Comment:
load data from json.  in exports. and render the data

Reply:
very well sir

Reply:
@vaibhavdaren sorry so say, I did not understand your concern. Can you please be more specific regarding the area that needs changes

Reply:
Yes have a file named data.json and render the array as if you are getting a response from API.
This is called data mocking. This practice is used when the frontend development is ahead of backend development in agile software development life cycle.

Reply:
Sure, Thank you , i will update it soon